### DISCUSSION TITLE
getFirstMedia - not showing responsive images

### DISCUSSION THREAD

**Discussion by SMSoumen** [spatie/laravel-medialibrary#3643] [Q&A]
getFirstMedia - not showing responsive images

**Comment by netzknecht** [comment]
If you haven't set the environment variable `QUEUE_CONNECTION` as `sync`, check if you [queue worker is running](https://laravel.com/docs/11.x/queues#the-queue-work-command). If the responsive images have not been generated, they will not be displayed.
